homeassistant/components/config/config_entries.py

```diff
@@ -372,6 +372,30 @@ async def ignore_config_flow(hass, connection, msg):
     connection.send_result(msg["id"])
 
 
+@websocket_api.require_admin
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        "type": "config_entries/remove_device",
+        "entry_id": str,
+        "device_id": str,
+    }
+)
+async def config_entry_remove_device(hass, connection, msg):
+    """Remove config entry from a device."""
+    result = await hass.config_entries.async_set_disabled_by(
```

Comment:
```suggestion
    result = await hass.config_entries.async_remove_device(
```

Reply:
How about `async_config_entry_will_remove_from_device` since we don't expect the config entry itself to remove itself from the device?

Reply:
Ok, but I think it's long.